applications/nrf_desktop/configuration/nrf52840_pca10056/app_ZDebug_dongle.conf

```diff
@@ -0,0 +1,170 @@
+#
+# Copyright (c) 2019 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+################################################################################
+# Application Configuration
+
+CONFIG_DESKTOP_INIT_LOG_MOTION_EVENT=n
+CONFIG_DESKTOP_INIT_LOG_HID_REPORT_EVENT=n
+CONFIG_DESKTOP_INIT_LOG_HID_REPORT_SENT_EVENT=n
+
+CONFIG_DESKTOP_HID_REPORT_DESC="configuration/common/hid_report_desc.c"
+
+CONFIG_DESKTOP_BUTTONS_ENABLE=y
+CONFIG_DESKTOP_BUTTONS_POLARITY_INVERSED=y
+
+CONFIG_DESKTOP_HID_DONGLE=y
```

Comment:
Click detector is necessary to control Bluetooth peers (ble_bond reacts on click event).
We could even add this as dependency.

Reply:
Added